src/index.js

```diff
@@ -1,7 +1,16 @@
 import low from 'lowlight'
 import visit from 'unist-util-visit'
 
-export default function attacher({include, exclude, prefix} = {}) {
+export default function attacher({include, exclude, prefix, languages} = {}) {
+
+  // register custom languages
+  if (languages) {
+    // eslint-disable-next-line guard-for-in
+    for (let name in languages) {
```

Comment:
or alternatively this could be done with `Object.entries` which also removes the prototype attributes https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries
```suggestion
    for (const [name, highlighter] of Object.entries(languages)) {
```